website/docs/fun/qotd/advanced/overview.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Overview
+---
+
+An advanced QOTD system which automatically posts a question every day, either a random default question or from a queue
+of suggested questions from your server members.
+
+## Features
+
+- Permits server members to add questions to a queue.
+  - Allows you to configure a queue channel to view the question queue and submit new questions.
+  - Allows you to delete any question from the queue at any time.
+- Posts a question in a delegated QOTD channel each day.
+  - Posts the oldest question in the queue.
+  - Posts a random default question if no questions in the queue.
+  - Allows you to force a new question post with `qotd push`.
+- Creates a new thread for each question.
+  - If you set up your QOTD channel as a forum channel, the system creates a new forum post for each question.
+- Publishes each question, if used in an announcement channel.
+- Pings a role on each new question.
+- Sticky message at the bottom of the QOTD channel reminding members what the question is.
+- Completely interactive configuration setup, no config variables required.
+
+:::info
+
+Most of these features are configurable via the interactive setup!
+
+:::
+
+## Installing
+
+As usual, there are instructions describing where to put the script and which trigger to use on the pages corresponding to the individual commands. Additionally, we've documented how and where to add these scripts down below.
+
+### Main Command
+
+Add the [main command](main-cc) as a new custom command. The trigger is a regex type with the trigger set to `\A`.
+
+### Component Handler
+
+Next, add the [component handler](component-handler) as a new custom command. The trigger is a component type with the
+Custom ID set to `\Aqotd-`.
+
+### Modal Handler
+
+Next, add the [modal handler](modal-handler) as a new custom command. The trigger is a modal type with the
+Custom ID set to `\A0?qotd-`.
+
+### Interval Trigger
+
+Finally, add the [interval command](interval) as a new custom command. The trigger is an hourly interval type with the
+interval set to `24` (or whatever interval you'd like). Set the `channel` menu to any channel. Then, disable the command
```

Comment:
```suggestion
Finally, add the [interval command](interval) as a new custom command. Configure this custom command to use the hourly interval trigger type, set to `24h` (or any other interval of your choice) in an arbitrary channel.
```
Tangentially, I think the comment re: disabling the command deserves a more prominent callout warning.

Reply:
Done both